rskj-core/src/main/java/org/ethereum/db/MutableRepository.java

```diff
@@ -327,18 +327,6 @@ public void syncTo(Trie root) {
         return rootHash.getBytes();
     }
 
-    // What's the difference between startTracking() and getSnapshotTo()?
```

Comment:
Until we have all the refactor in place, I would not lose the information in this comment. I would probably not keep it here, but maybe we can move the relevant parts into the javadoc of each method (I mean `co.rsk.db.RepositoryLocator#snapshotAt` and `org.ethereum.core.Repository#startTracking`)

Reply:
I don't see anything important in the comment, and also our next refactors (e.g remove `syncToRoot` will make this transparent 😄)